MineSweeper.py

```diff
@@ -0,0 +1,256 @@
+import Tkinter as tk
+import random
+
+import sys
+sys.path.append('~/Library/Python/2.7/site-packages')
+
+from drawingpanel import *
+
+BOARD_HEIGHT = 300
+BOARD_WIDTH = 300
+BOX_SIZE = 20
+
+NUMBER_MINES = ((BOARD_WIDTH / BOX_SIZE - 1)**2)//10 
+
+class Game():
+
+  def __init__(self):
+    self.master = tk.Tk()
+    self.panel = Canvas(self.master, width = BOARD_WIDTH, height = BOARD_HEIGHT + 100, bg = 'white')
+    self.panel.pack()
+
+
+     
+  def game_over(self):
+
+    self.master.unbind("<Button-1>")
+    self.master.unbind("<Button-2>")
+
+    self.b = Button(self.master, text = 'Play Again?', command = self.new_game)
+    self.panel.create_text(BOARD_WIDTH/2, BOARD_HEIGHT + 40, text  = 'Game Over', fill = 'red') 
+    self.b.pack()
+
+  def new_game(self):
+
+    self.master.destroy()
+
+    main()
+
+
+class Board(Game):
+
+  def __init__(self, Game):
+   
+    self.game = Game
+
+    self.__length = BOARD_HEIGHT/BOX_SIZE - 1
+    self.__width = BOARD_WIDTH/BOX_SIZE -1
+
+    self.__tiles = []
+    
+    row = []
+
+    for i in range(self.__length):
+      for j in range(self.__width):
+        new_tile = Tile(i*BOX_SIZE, j*BOX_SIZE, BOX_SIZE)
+        row.append(new_tile)
+
+      self.__tiles.append(row)
+      row = []
+
+
+  def draw(self, panel):
+
+    panel.delete(ALL)
+
+    marked_mines = 0
+
+    for row in self.__tiles:
+      for tile in row:
+        tile.draw(panel)
+      
+        if(tile.marked_mine):
+          marked_mines += 1
+
+    mines = NUMBER_MINES - marked_mines
+
+    bottom_text = 'Mines left: ' + repr(mines)
```

Comment:
it's full name is represent. It makes whatever it is called on into what the interpreter is expecting. In this case turning an integer into a string